### PR TITLE
Trello: Added search your cards feature

### PR DESCRIPTION
**Pull request by ajaypremshankar**
## Description

Now you can search and open your cards from trello.com

## Screencast
Screenshot 2022-03-24 at 11 50 24 PM

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed in the `metadata` folder

### CODE REVIEW

File: extensions/trello/src/utils/fetchTodos.tsx [hunk 1]
```diff
@@ -29,12 +30,12 @@ export const returnTodos = async (searchTerm: string): Promise<TrelloFetchRespon
         const json = await response.json();
         return json as TrelloFetchResponse;
       } catch (error) {
-        showToast(ToastStyle.Failure, "An error occured", "Could not fetch todos, check your credentials");
```
Comment (ajaypremshankar) [inline extensions/trello/src/utils/fetchTodos.tsx:32]: Removed usage of deprecated `ToastStyle`

File: extensions/trello/src/searchTodos.tsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
-import { List, showToast, ToastStyle } from "@raycast/api";
-import { useEffect, useState } from "react";
```
Comment (ajaypremshankar) [inline extensions/trello/src/searchTodos.tsx:2]: Got rid of unused imports.

File: extensions/trello/package.json [hunk 3]
```diff
@@ -15,11 +15,18 @@
       "mode": "view"
     },
     {
-      "name": "search",
```
Comment (ajaypremshankar) [inline extensions/trello/package.json:18]: Named appropriately.

**Comment by ChrisChinchilla** [comment]
@pernielsentikaer and @ajaypremshankar yup, it works fine and as expected. I think this will be useful later too, to bolt various functions together.

**Pull request merged**